src/applications/proxy-rewrite/partials/mobile/header/index.js

```diff
@@ -0,0 +1,152 @@
+import { makeMegaMenu } from './mega-menu';
```

Comment:
Plain HTML version of the header minus mega menu which is in a different file (used for mobile)